src/app/card/card.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component, OnInit, Input, Output, EventEmitter, OnDestroy } from '@angular/core';
+import { Router, ActivatedRoute } from '@angular/router';
+import { Broadcaster, Logger } from 'ngx-base';
+
+import { WorkItem } from './../models/work-item';
+
+@Component({
+  selector: 'card',
+  templateUrl: './card.component.html',
+  styleUrls: ['./card.component.scss']
+})
+export class CardComponent implements OnInit {
+
+  @Input() item: WorkItem;
```

Comment:
Input should not be just work item type data, it can be anything. Card should have it's own format. Like, title, description, logo, menuItems etc. We need to make it generic so that we can use it in multiple places. 
@michaelkleinhenz @nimishamukherjee thoughts ?